recipe/meta.yaml

```diff
@@ -26,20 +24,20 @@ requirements:
     - wget
     - llvm-openmp  # [osx]
     - libgomp      # [linux]
+    - libtool
   host:
     - python
     - pip
     - gsl
     - fftw
     - cfitsio
-    - libsharp
     - chealpix
     - numpy
+    - libtool
```

Comment:
Just an FYI, a build tool like this never goes in host.

Reply:
Ah, thanks a lot. I wasn't sure from the description I found online.